diagnostic.md

```diff
@@ -5,74 +5,87 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back-end or  "server-side", is essential to a functioning site and controls updates and changes. Essentially everything the user can't see in the browser such as databases and servers.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+I am not sure, here
```

Comment:
Because we build our own Client 'views' with HTML/Handlebars/Javascript etcetera.